### ISSUE TITLE
[reproduce] SimCLR Model Training - BigEarthNet Data

### ISSUE THREAD

**Issue by taeil** [Berkeley-Data/irrigation_detection#24]
Follow the setup instructions in the Readme in the Setup Folder to install the docker container.
Run the docker container interactively passing in the mounted files from cloud storage:
nvidia-docker run -it --rm -v /mnt/irrigation_data:/data irgapp bash
From within the docker container, copy the necessary clouds from cloud storage to the /root/capstone_fall20_irrigation/BigEarthData/tfrecords directory
The #6 command command will place you within the docker container. Train the model using the following:
nohup python simclr.py -a ARCH -o OUTPUT -e EPOCHS -b BATCH -t TEMPERATURE -c CALIFORNIA& where ARCH is 'InceptionV3', 'ResNet50', 'Xception', or 'ResNet101V2' OUTPUT is a prefix for model file and results file EPOCHS is number of epochs to run (50 is default) BATCH is batch size (default is 32). TEMPERATURE is the temperature used in the contrastive loss function (default 0.1) CALIFORNIA is True or False and specifies whether to train on California data (True) or BigEarthNet data (False).
The above command will run the python script in background which will allow for grabbing model files as soon as they are saved. You can watch the progress by using the command below.
tail -f nohup.out

**Comment by suryatechie** [comment]
I will work on this task.
